frontend/src/components/FollowersList.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
```

Comment:
I think you could refactor FollowersList.js and FollowingsList.js into 1 component since they are so similar and just pass different data into them, kinda like what we do with Card.js